examples/crypto.rs

```diff
@@ -0,0 +1,59 @@
+use pubnub::core::CryptoProvider;
+use pubnub::providers::crypto::CryptoModule;
+use pubnub::{Keyset, PubNubClientBuilder};
+use std::env;
+
```

Comment:
it would be nice to have some kind of description that user can encrypt data partially (the first part of the example) and fully automated (the second one).